tools/pctools/Dockerfile

```diff
@@ -0,0 +1,53 @@
+FROM registry.suse.com/bci/python:3.6
+
+RUN [ "cat", "/etc/os-release" ]
+
+ARG WKDIR=/home/venv
+
+# this dir will be a shared volume at runtime
+ARG SHARDIR=/home/tmp
+
+# script to install
+ARG VENV=./venv_inst.sh
+
+# venv dir and pip modules to install:
+# AWS
+ARG DIRAWS=aws 
+ARG OUTAWS="resources_$DIRAWS.txt" 
```

Comment:
Why is here another variable (`$DIRAWS`) being used?

This gets a bit too convoluted. We are using an `ARG`, which contains a bash variable, which is passed as an argument to a bash script ...  
Is this level of complexity really necessary? Following the flow is difficult here.

Reply:
I was using the variable to create a filename with a proper common template for the copy phase. The same variable is ised  to create a venv dir. But I agree, I can change somethings and reduce parameters use